includes/core/classes/class-event-setup.php

```diff
@@ -74,6 +75,151 @@ protected function setup_hooks(): void {
 		add_filter( 'get_the_date', array( $this, 'get_the_event_date' ) );
 		add_filter( 'the_time', array( $this, 'get_the_event_date' ) );
 		add_filter( 'display_post_states', array( $this, 'set_event_archive_labels' ), 10, 2 );
+
+		/**
+		 * WHERE SHOULD ALL THAT GO?
+		 *
+		 * @todo Refactor into a
+		 *       class-add-to-calendar.php
+		 *       or another DRY place ?!
+		 */
+		add_action(
+			'init',
+			function () {
```

Comment:
Let's avoid anonymous functions in hooks. They are quite hard to debug and test.